tools/gatk_getpileupsummaries.cwl

```diff
@@ -27,7 +27,7 @@ inputs:
   reference: File
   interval_list: File
   exac_common_vcf: {type: File, secondaryFiles: [.tbi]}
-  max_memory: {type: int?, default: 2000, doc: "Maximum memory in MB for GATK GetPileupSummaries to use"}
+  max_memory: {type: int?, default: 20, doc: "Maximum memory in GB for GATK GetPileupSummaries to use"}
```

Comment:
hmmm, this went from 2000 MB (2GB) to 20GB....intentional?

Reply:
Nope. Just bad maths.

Reply:
Fixed.